docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
       - postgres
   postgres:
     container_name: postgres
-    image: postgres:12-alpine3.18
+    image: postgres
```

Comment:
Why dropping alpine and versions? It does make sense to define them to know what version of Postgres is running and Alpine to keep images small...

Reply:
I had an issue running it with `pgAdmin`

Reply:
Could it be that there was an existing database data storage which was made in a different postgresql version?
At least I had those issues before defining a specific version...

Reply:
Added a commit to specify also the pgadmin version and link where to find the supported postres versions. Tested with fresh build and "works for me"